src/site/antora/modules/ROOT/examples/manual/migration/Migration1Example.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package example;
+
+import org.apache.log4j.Logger;
+
+public class Migration1Example {
+
+    private static final Logger logger = Logger.getLogger(Migration1Example.class);
```

Comment:
Could you rename all `static final`s to upper-case, please? That is, `s/logger/LOGGER/`. Note that this is the recommended practice in `getting-started.adoc` too.

Reply:
Fixed in 82a6f08f9aadc6043d5594fac9bf7589912c676b